src/Distributions.jl

```diff
@@ -239,6 +240,7 @@ export
     quantile,           # inverse of cdf (defined for p in (0,1))
     qqbuild,            # build a paired quantiles data structure for qqplots
     rate,               # get the rate parameter
+    excessprob,         # get the exess probability of zeros parameter (ZeroInflatedPoison)
```

Comment:
I am not sure if we should introduce and in particular export a new function here just for `ZeroInflatedPoisson`.